dillorc

```diff
@@ -378,7 +378,11 @@ ui_tab_bg_color=#b7beb7
 
 # A mouse's middle click over a tab closes the Tab.
 # With mousewheel mouses, right click feels way better (set to YES).
-#right_click_closes_tab=NO
+#right_click_closes_tab=YES
+
+# Use the mouse wheel to switch tabs.
+# If set to NO, the page will be scrolled instead.
+#mouse_wheel_switches_tabs=YES
```

Comment:
Maybe a better name would be *scroll_switches_tabs*? As by "mouse wheel" is not clear if we mean clicking the mouse wheel or scrolling it. Also, some input methods (like touchpad) don't really have a wheel.